client/rpcserver/handlers.go

```diff
@@ -301,86 +308,108 @@ func sortHelpKeys() []string {
 	return keys
 }
 
-// helpMsgs are a map of routes to help messages. The first string is a short
-// description of usage. The second is a brief description and a breakdown of
-// the arguments and return values.
-var helpMsgs = map[string][2]string{
-	helpRoute: {`("cmd")`,
-		`Print a help message.
-
-Args:
+// helpMsgs are a map of routes to help messages. They are broken down into six
+// sections.
+// In descending order:
+// 1. Password argument example inputs. These are arguments the caller may not
+//    want to echo listed in order of input.
+// 2. Argument example inputs. These are non-sensitive arguments listed in order
+//    of input.
+// 3. A description of the command.
+// 4. An extensive breakdown of the password arguments.
+// 5. An extensive breakdown of the arguements.
+// 6. An extensive breakdown of the returned values.
+var helpMsgs = map[string][6]string{
```

Comment:
With 6 categories here now, I think it's worth using a `struct` with named fields.